test/exercises_test.dart

```diff
@@ -1,5 +1,3 @@
-@Timeout(const Duration(seconds: 120))
```

Comment:
I removed this because it was causing the tests to time out every single time we ran this set.

Now, when people add new tests, we won't need to remember to update this.

Reply:
Without it, we were getting timeouts, so I'm interested in the impact.